From 227e2bed0f0b576c5897aff3cef977d1219bb211 Mon Sep 17 00:00:00 2001
From: fourmone <eric.fourmont-ext@st.com>
Date: Thu, 9 Jan 2025 17:12:04 +0100
Subject: [PATCH] arm64: dts: st: correct imx335 reset gpio polarity in
 stm32mp235f-dk.dts

    With commit 9eac174a9b89 ("media: imx335: Fix reset-gpio handling")
    introduction, reset gpio line must be corrected within the device-tree
    in order to match with expected behavior of the reset of module.

Change-Id: Iaf55991df58930717ab5b0db04b2815684bd6108
Signed-off-by: fourmone <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/427894
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp235f-dk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp235f-dk.dts
@@ -323,7 +323,7 @@
 		avdd-supply = <&imx335_2v9>;
 		ovdd-supply = <&imx335_1v8>;
 		dvdd-supply = <&imx335_1v2>;
-		reset-gpios = <&gpiob 1 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
+		reset-gpios = <&gpiob 1 (GPIO_ACTIVE_LOW | GPIO_PUSH_PULL)>;
 		powerdown-gpios = <&gpiob 11 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
 		status = "okay";
 
